release.sh

```diff
@@ -51,6 +51,26 @@ function assert_changelog_present() {
   fi
 }
 
+function assert_version_updated() {
+  # Verify that the vagrant-spk version has been updated.
+  EXPECTED_SCRIPT_VERSION="__version__ = \"$TAG_NAME\""
+  if [ "$(head -n 26 vagrant-spk | tail -n 1)" != "$EXPECTED_SCRIPT_VERSION" ]; then
```

Comment:
Could we not just run `vagrant-spk --version` here?

Reply:
Yeah, we can.